src/HL/View/Home/Features.hs

```diff
@@ -56,11 +54,7 @@ purefunc =
 statically :: Html ()
 statically =
   do h2_ "Statically typed"
-     p_ "Every expression in Haskell has a type which is determined at compile time. \
-       \All the types composed together by function application have to match up. If \
-       \they don't, the program will be rejected by the compiler. Types become not \
-       \only a form of guarantee, but a language for expressing the construction \
-       \of programs."
+     p_ ""
```

Comment:
Removing this text entirely will make the page look unbalanced.  I'd prefer some useful text than nothing. Please don't delete any sections without having something better to replace it. ☝️🐵